utils/spdx/src/test/kotlin/UtilsTest.kt

```diff
@@ -166,6 +166,13 @@ class UtilsTest : WordSpec() {
             "return null for an unknown SPDX LicenseRef" {
                 getLicenseText("LicenseRef-foo-bar") should beNull()
             }
+
+            "return the full license text for a LicenseRef-ort license" {
+                val text = getLicenseText("LicenseRef-ort-oracle-futc")?.trim()
```

Comment:
I believe we should not trim (at least from the end) here, but ensure that files are committed without trailing spaces. Also trimming from the start is not required as there is not leading space before the asserted "License Rights and Restrictions".

Reply:
This was done for consistency with other similar tests, see lines 139 and 150.